projects/samples/contests/robocup/controllers/referee/referee.py

```diff
@@ -2760,10 +2771,9 @@ def read_team(json_path):
             ball_holding = check_ball_holding()       # check for ball holding fouls
             if ball_holding:
                 interruption('FREEKICK', ball_holding, game.ball_position)
-            ball_handling = check_ball_handling()
+            ball_handling = check_ball_handling()  # return team id if ball handling is performed by goalkeeper
```

Comment:
One of the thing I'm not sure with this is:
- Where is ball handling from the other players checked?